### DISCUSSION TITLE
ScreenToGif 2.35.2

### DISCUSSION THREAD

**Discussion by NickeManarin** [NickeManarin/ScreenToGif#1029] [Announcements]
# What's new?

• If this installer fails to update from v2.34.1 or older, download and run it manually.  
• Just bug fixes.  

### Fixed:

♦ When applying the smooth loop feature from the start, a warning message was appearing, but frames were being removed anyway.  
♦ The install process was still failing if you had SharpDx set up before (error 2711).  

### More info:

[Known issues](https://github.com/NickeManarin/ScreenToGif/wiki/Help-▪-Known-issues)

<hr /><em>This discussion was created from the release <a href='https://github.com/NickeManarin/ScreenToGif/releases/tag/2.35.2'>ScreenToGif 2.35.2</a>.</em>

**Comment by vape2019** [comment]
3 point releases in 5 hours... you are starting to get like Mozilla and Google with your release frequency.

**Reply by NickeManarin** [reply]
Issues with releasing an enormous set of changes:
• Updated version of framework (.NET 6).
• Separated binaries for each architecture (x86, x64 and arm64).
• Updated installer logic to handle missing requirements.
• Updated the update system to get the correct installer during update.

Just that to have one thing break :(

The planned cadence of releases is set to be at a maximum of 1.25 per month, unless something important breaks.
